worker/config/config.go

```diff
@@ -127,6 +134,20 @@ func (c *Config) Defaults() error {
 		return fmt.Errorf("must specify local registry directory")
 	}
 
+	// mem usage limits for creating sandboxes
+	if c.Create_sandbox_soft_limit == 0.0 {
+		c.Create_sandbox_soft_limit = 0.5
```

Comment:
Can't we push the default soft limit closer than this?  Say 0.8?

Reply:
We can certainly do this.